SQL/0000-00-00-schema.sql

```diff
@@ -67,7 +67,9 @@ CREATE TABLE `language` (
   UNIQUE KEY (`language_code`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
-INSERT INTO language (language_code, language_label) VALUES ('en-CA', 'English');
+INSERT INTO language (language_code, language_label) VALUES
+    ('en-CA', 'English'),
```

Comment:
It probably makes sense to only provide `en-US` or `en-CA` as the default language and not `fr-CA`, since we don't support French (canadian or other variant) anywhere in LORIS